android-emulator-plugin/src/main/java/com/quittle/androidemulator/AndroidEmulatorPlugin.java

```diff
@@ -65,18 +65,22 @@ private static void setUpAndroidTests(final Project project) {
                 });
     }
 
+    private static boolean ensureFileIsExecutable(final File file) {
```

Comment:
Can you add JavaDoc to this method, explaining why it is necessary so it doesn't get accidentally removed in the future?